hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/types/NamespaceSummaryResponse.java

```diff
@@ -43,6 +43,16 @@ public class NamespaceSummaryResponse {
   @JsonProperty("numKey")
   private long numTotalKey;
 
+  /** Age of entity in
+   * human-readable format. */
+  @JsonProperty("age")
+  private long age;
```

Comment:
can have same name as createTime, UI can calculate age and display name accordingly

Reply:
@sumitagrawl - Thanks for reviewing. It has been fixed. Pls re-review.